src/test/integration/python_test_files/file_7.py

```diff
@@ -0,0 +1,10 @@
+from __future__ import annotations
+
+
+def function(
+    arg1: int,
+    arg2: list[str] | dict[str, int] | Thing,
+    kwarg1: int = 1
+) -> Generator[tuple[str, str]]:
```

Comment:
Can you change the return type to also use the PEP 608 style just to be extra sure everything works?

Reply:
Good point this needed a change in a different place than the tokenizer 👍